### ISSUE TITLE
Questions about training targets

### ISSUE THREAD

**Issue by isseebx123** [lightvector/KataGo#258]
First of all, thank you for the great experiments.

I am curious about training targets that are not described in the paper.
Can you tell me about td_value, variance_time, lead, scoring, seki, futurepos? 
I also want to know what effects these training targets have and what you expected. 

Sorry for asking too many questions at once. 
If you've ever described it somewhere, I would really appreciate it if you give me a link.

**Comment by lightvector** [comment]
Thank you too! I've probably described it in discord chat, but nowhere really organized.

* variance_time - Unused

* td_value - Trained to predict the short-term future MCTS winrate. Short-term future means the an exponentially weighted sum of near future winrates, averaging in expectation from about 6 moves to about 60 moves in the future. The idea is that maybe we can make use of training the net on something less noisy than the final game outcome for when it makes tactical blunders that it realizes not too long in the future, later, but only as auxiliary targets, leaving the primary value head still the same as in AlphaZero, which I'm fairly sure is important for Go.

I think these targets may have a mild benefit, but nothing major. But they're basically free, it costs nothing additional to write down and compute.

* lead - Predict the number of komi that must be added/subtracted required for the net to believe the game is fair (50% winrate). This was introduced because this is a MUCH better value to display for Go players. When Go players ask "how many points am I ahead" what they *mean* is "how many points am I in excess of an even game?", they do not mean "if we played this game out how many points would I win by on average" because the latter may be hugely biased by the way each player may take or avoid risks (giving up points relative to optimal) based on whether they're winning or not.

* scoring - This is almost the same as ownership, except that has some adjustments for Japanese seki and/or group tax rules. It's intended that the sum of scoring, plus adjustments such as komi and white bonus points, equals the final score of the game. This was not true of how ownership was calculated. Possibly ownership should have just been replaced with this target, in any case, this aux target is only needed because of trying to support Japanese rules.

* seki - These are a few aux targets intended to help the neural net recognize seki positions for Japanese rules, which was a problem early on in training. The net must predict stone that will end the game being in seki, as well as territories that under seki rules would be scored differently. 

* futurepos -  This one is just having the net predict the future position, 8, 16, 32 moves in the future. It may add a small benefit to the net for regularization, but also not a big deal.

So basically, td_value and futurepos are intended to improve the net's learning just like other aux targets, scoring and seki are to help Japanese rules learning and and might not be needed in any case but at least that was the intention, and lead is because "scoreMean" is highly biased and inadequate for helping Go players analyze games, lead is much better to display.
